internal/wasm/memory_test.go

```diff
@@ -733,6 +733,13 @@ func TestMemoryInstance_Write(t *testing.T) {
 
 	ok = mem.Write(9, buf)
 	require.False(t, ok)
+
+	// Test overflow
+	huge := uint64(math.MaxUint32 + 1 + 4)
+	if huge == uint64(int(huge)) { // Only run test on 64-bit platforms
+		ok := mem.Write(4, make([]byte, huge))
```

Comment:
actually I'm now wondering if we should make a byte array so big in unit tests :P

Reply:
should we add another dimension to TestMemoryInstance_HasSize instead?

Reply:
On 64-bit shouldn't be a problem (at least on Linux?): memory isn't committed, just reserved.

But let me try to avoid the issue and create a pretend slice instead.

Reply:
cool, I think we would end up running this on win, darwin, and linux. One easy enough way is to look at before/after time if any difference, then add a comment like `// don't worry: memory isn't committed, just reserved.`